addons/sale_product_configurator/static/src/js/sale_product_field.js

```diff
@@ -148,12 +118,13 @@ patch(SaleOrderLineProductField.prototype, {
                         position: 'bottom',
                         mode: "readonly",
                     });
-                    await applyProduct(line, optionalProduct);
+                    await this.applyProduct(line, optionalProduct);
                 }
             },
             discard: () => {
                 saleOrderRecord.data.order_line.delete(this.props.record);
             },
+            options: this.props.options,
```

Comment:
If we still have a separate class & widget for sale, can't we hardcode the options here (show_qty & co) instead of defining them in the view?
It's not something we want to allow the customer to customize anyway.